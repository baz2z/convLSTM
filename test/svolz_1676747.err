slurmstepd: error: *** JOB 1676747 ON slurm-bm-60 CANCELLED AT 2022-08-04T10:45:03 ***
