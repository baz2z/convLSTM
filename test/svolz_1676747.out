/mnt/qb/work/butz/svolz67/dev/convLSTM/test
JOB DATA
JobId=1676747 ArrayJobId=1676747 ArrayTaskId=1 JobName=review.sh
   UserId=svolz67(4458) GroupId=butz(4032) MCS_label=N/A
   Priority=20054 Nice=0 Account=butz QOS=normal
   JobState=RUNNING Reason=None Dependency=(null)
   Requeue=1 Restarts=0 BatchFlag=1 Reboot=0 ExitCode=0:0
   RunTime=00:00:00 TimeLimit=3-00:00:00 TimeMin=N/A
   SubmitTime=2022-08-04T10:44:48 EligibleTime=2022-08-04T10:44:49
   AccrueTime=Unknown
   StartTime=2022-08-04T10:44:49 EndTime=2022-08-07T10:44:49 Deadline=N/A
   PreemptEligibleTime=2022-08-04T10:45:49 PreemptTime=None
   SuspendTime=None SecsPreSuspend=0 LastSchedEval=2022-08-04T10:44:49
   Partition=gpu-2080ti AllocNode:Sid=bg-slurmb-login1:29283
   ReqNodeList=(null) ExcNodeList=(null)
   NodeList=slurm-bm-60
   BatchHost=slurm-bm-60
   NumNodes=1 NumCPUs=8 NumTasks=1 CPUs/Task=1 ReqB:S:C:T=0:0:*:*
   TRES=cpu=8,mem=3G,node=1,billing=2,gres/gpu=1
   Socks/Node=* NtasksPerN:B:S:C=0:0:*:* CoreSpec=*
   MinCPUsNode=1 MinMemoryNode=3G MinTmpDiskNode=0
   Features=(null) DelayBoot=00:00:00
   OverSubscribe=OK Contiguous=0 Licenses=(null) Network=(null)
   Command=/mnt/qb/work/butz/svolz67/dev/convLSTM/test/review.sh
   WorkDir=/mnt/qb/work/butz/svolz67/dev/convLSTM/test
   StdErr=/mnt/qb/work/butz/svolz67/dev/convLSTM/test/svolz_1676747.err
   StdIn=/dev/null
   StdOut=/mnt/qb/work/butz/svolz67/dev/convLSTM/test/svolz_1676747.out
   Power=
   CpusPerTres=gpu:8
   TresPerNode=gpu:1
   MailUser=sebastian.volz@student.uni-tuebingen.de MailType=FAIL
   NtasksPerTRES:0

node:
slurm-bm-60.sdn
RUN Script
